app/models/beneficiaire.rb

```diff
@@ -13,6 +13,10 @@ class Beneficiaire < ApplicationRecord
   scope :par_date_creation_asc, -> { order(created_at: :asc) }
   scope :sauf_pour, ->(id) { where.not(id: id) }
 
+  def evaluations_diagnostic
```

Comment:
on peut peut-être utiliser un delegate ? `delegate :diagnostic, to: :evaluations, prefix: true`